collector/index.md

```diff
@@ -0,0 +1,425 @@
+    </tr>
+    <tr>
+      <td>db_username (DB_USERNAME)</td>
+      <td>n/a - either this or db_url is required</td>
+      <td>user to connect as</td>
+    </tr>
+    <tr>
+      <td>db_password (DB_PASSWORD)</td>
+      <td>n/a - either this or db_url is required</td>
+      <td>password to connect with</td>
+    </tr>
+    <tr>
+      <td>db_host (DB_HOST)</td>
+      <td>n/a - either this or db_url is required</td>
+      <td>host to connect to</td>
+    </tr>
+    <tr>
+      <td>db_port (DB_PORT)</td>
+      <td>n/a - either this or db_url is required</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>db_sslmode (DB_SSLMODE)</td>
+      <td>prefer</td>
+      <td>sslmode to connect with (see <a target="_blank" href="https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNECT-SSLMODE">Postgres documentation</a> for more details)</td>
+    </tr>
+    <tr>
+      <td>db_sslrootcert (DB_SSLROOTCERT)</td>
+      <td>system certificate store (?)</td>
+      <td>path to SSL certificate authority (CA) certificate(s) to use to verify the server's certificate, or one of <code>rds-ca-2015-root</code> or <code>rds-ca-2019-root</code> to use the built-in AWS RDS certificates</td>
+    </tr>
+    <tr>
+      <td>db_sslrootcert_contents (DB_SSLROOTCERT_CONTENTS)</td>
+      <td>n/a, see above</td>
+      <td>as above, but actual contents of the certificate(s)</td>
+    </tr>
+    <tr>
+      <td>db_sslcert (DB_SSLCERT)</td>
+      <td>[none]</td>
+      <td>file name of the client SSL certificate</td>
+    </tr>
+    <tr>
+      <td>db_sslcert_contents (DB_SSLCERT_CONTENTS)</td>
+      <td>[none]</td>
+      <td>as above, but actual contents of the certificate</td>
+    </tr>
+    <tr>
+      <td>db_sslkey (DB_SSLKEY)</td>
+      <td>[none]</td>
+      <td>location for the secret key used for the client certificate</td>
+    </tr>
+    <tr>
+      <td>db_sslkey_contents (DB_SSLKEY_CONTENTS)</td>
+      <td>[none]</td>
+      <td>as above, but actual contents of the key</td>
+    </tr>
+  </tbody>
+</table>
+
+## AWS settings
+
+Only relevant if you are running your database in AWS RDS. Note that the <code>aws_endpoint_*</code>
+settings are only relevant if you are using custom AWS endpoints. See
+[the AWS documentation](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/) for details.
+
+<table>
+  <thead>
+    <tr>
+      <th>setting</th>
+      <th>default</th>
+      <th>description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>aws_region (AWS_REGION)</td>
+      <td>n/a, required if aws</td>
+      <td>region your AWS server is running in</td>
+    </tr>
+    <tr>
+      <td>aws_db_instance_id (AWS_INSTANCE_ID)</td>
+      <td>n/a, required if aws</td>
+      <td>the instance id of your AWS server</td>
+    </tr>
+    <tr>
+      <td>aws_access_key_id (AWS_ACCESS_KEY_ID)</td>
+      <td>n/a, required if aws and using logs?</td>
```

Comment:
This is not needed when using instance roles (which we can get from the metadata service when using an EC2 instance)